internal/website/content/howto/docstore/actions.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Actions"
+date: 2019-06-11T10:11:59-04:00
+draft: true
+weight: 2
+---
+
+In Docstore, you use actions to read, modify and write documents. You can
+execute a single action, or run multiple actions together in an _action list_.
+
+<!--more-->
+
+## Representing Documents
+
+We'll use a collection with documents represented by this Go struct:
+```
```

Comment:
nit: blank line before code block and use ```` ```go ```` to highlight with Go syntax. (Here and elsewhere.)

````suggestion

```go
````

Reply:
Done.